recipes/box2d/all/conanfile.py

```diff
@@ -39,15 +50,41 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def requirements(self):
+        if Version(self.version) >= "3.0.0":
+            self.requires("simde/0.8.2")
+
+    def validate(self):
+        if Version(self.version) < "3.0.0":
+            return
+
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
```

Comment:
Note that once we drop Conan 1 support, Conan 2 added support for `compiler.cstd`, which could be useful for C projects like this, see https://docs.conan.io/2/reference/tools/build.html#cstd